--- SuiteSparse_config/SuiteSparse_config.mk.old	2015-01-04 11:09:28.581838569 +0100
+++ SuiteSparse_config/SuiteSparse_config.mk	2015-01-04 13:59:59.606264602 +0100
@@ -95,8 +95,8 @@
 # LIB = -lm
 
 # For "make install"
-INSTALL_LIB = /usr/local/lib
-INSTALL_INCLUDE = /usr/local/include
+INSTALL_LIB = $(LIBRARY_PATH)
+INSTALL_INCLUDE = $(INCLUDE_PATH)
 
 # Which version of MAKE you are using (default is "make")
 # MAKE = make
