diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5ab15f5..95426fa 100644
--- CMakeLists.txt
+++ CMakeLists.txt
@@ -399,14 +399,14 @@ endif()
 
 # Check for UMFPACK
 if (DOLFIN_ENABLE_UMFPACK)
-  find_package(AMD QUIET)
-  find_package(BLAS QUIET)
-  find_package(UMFPACK QUIET)
+  find_package(AMD REQUIRED)
+  find_package(BLAS REQUIRED)
+  find_package(UMFPACK REQUIRED)
 endif()
 
 # Check for CHOLMOD
 if (DOLFIN_ENABLE_CHOLMOD)
-  find_package(CHOLMOD QUIET)
+  find_package(CHOLMOD REQUIRED)
 endif()
 
 # Check for HDF5
